split.py

```diff
@@ -80,16 +80,27 @@ def parse_file_start(split_file):
     return split_file[0] if "start" not in split_file else split_file["start"]
 
 
-def gather_c_funcs(repo_path):
+def get_symbol_addrs_path(repo_path, options):
+    if "symbol_addrs" in options:
```

Comment:
I'd use options.get() here with the fallback value being "symbol_addrs.txt" (see other examples in the codebase). Otherwise, lgtm!

Reply:
oh, and same for the undefined syms path

Reply:
what about the "tools" part?

Reply:
sorry yeah I meant just what's there already, though I think maybe the default values should change at some point

Reply:
nm remove tools